### PR TITLE
Update import_template.ini

### PR DESCRIPTION
**Pull request by ab-e**
add uri_postfix to fix reading of xml from nerc website

### CODE REVIEW

File: harvester.py [hunk 1]
```diff
@@ -50,8 +52,9 @@ def read_xml(terminology):
                     with open(file_abs_path, 'wb') as f:
                         f.write(req_main.content)
                     # write down the corresponding ETag of a collection into .ini file
-                    header_ETag = head.headers['ETag']
-                    add_config_ETag(config_file_name, collection_name, header_ETag)
+                    #header_ETag = head.headers['ETag']
```
Comment (uschindler) [inline harvester.py:55]: could you revert this change, as this disables incremental harvesting?

File: harvester.py [hunk 2]
```diff
@@ -20,10 +20,12 @@ def read_xml(terminology):
     OUT: ET root object
     '''
     url = terminology['uri']
+    uri_postfix = read_config_uriPostfix(config_file_name)
+    url = url + uri_postfix
     collection_name = terminology['collection_name']
     try:
         head = requests.head(url)
-        if head.headers['Content-Type'] == 'application/rdf+xml':
+        if head.headers['Content-Type'] == 'application/rdf+xml; charset=utf-8':
```
Comment (uschindler) [inline harvester.py]: this should maybe more flexible, especially with whitespace. see the next one below!
I'd suggest to just do "starts with application/rdf+xml"

File: harvester.py [hunk 3]
```diff
@@ -50,8 +52,9 @@ def read_xml(terminology):
                     with open(file_abs_path, 'wb') as f:
                         f.write(req_main.content)
                     # write down the corresponding ETag of a collection into .ini file
-                    header_ETag = head.headers['ETag']
-                    add_config_ETag(config_file_name, collection_name, header_ETag)
+                    #header_ETag = head.headers['ETag']
+                    #add_config_ETag(config_file_name, collection_name, header_ETag)
+                    xml_content = req_main.content
 
         elif head.headers['Content-Type'] == 'text/xml;charset=UTF-8':
```
Comment (uschindler) [inline harvester.py]: see above!

**Review by uschindler — CHANGES_REQUESTED** [review]

File: harvester.py (hunk 1 above)
Comment (ab-e) [reply harvester.py:55]: when enabled, code fails to run because ETAg / etag seems to have been stripped from the header (KeyError: 'etag')

File: harvester.py (hunk 2 above)
Comment (ab-e) [reply harvester.py]: will do

Comment (ab-e) [reply harvester.py]: done.
